spec/Cabin/Seat.vspec

```diff
@@ -70,6 +82,25 @@ MassageLevel:
   unit: percent
   description: Seat massage level. 0 = off. 100 = max massage.
 
+MassageStatus:
+  datatype: string
+  type: sensor
+  allowed: ['ON', 'OFF']
+  description: Massage status.  
+
+SupportedMassageTypes:
+  datatype: string[]
+  type: attribute
+  allowed:  ['ROLL','KNEAD','ANTI FATIGUE','PULSE','STRETCH','LUMBAR','SHOULDER','WAVE','RELIEF','GLIDE']
```

Comment:
One error reported by CI concerns the `vss2ddsidl` exporter. In short it cannot handle whitespace in identifiers, like `ANTI FATIGUE`. It is valid VSS though, so one could see it as a defect in the exporter as VSS allowed values NOR are the same as enums. Anyhow, I suggest just changing the name to `ANTI_FATIGUE` to follow classic enum syntax.

```
Successfully installed cyclonedds-0.10.5
/home/runner/work/vehicle_signal_specification/vehicle_signal_specification/vss.idl:3828:50: syntax error
/home/runner/work/vehicle_signal_specification/vehicle_signal_specification/vss.idl:3852: fatal error: File write error
    };
```